editor/scene_tree_dock.cpp

```diff
@@ -1705,7 +1708,10 @@ void SceneTreeDock::_create() {
 			ERR_FAIL_COND(!newnode);
 
 			replace_node(n, newnode);
+
```

Comment:
Clang-format says this empty line before the `}` should be removed. https://travis-ci.org/godotengine/godot/jobs/439926660#L534

Reply:
I saw that yesterday before going to bed, fixed it, amended the commit and pushed, saw tarvis start up again so I turned my computer off and went to bed. So when I read this comment this morning I thought "yeah, I already fixed it", but I saw tarvis was still broken, and the line was still there... Apparently I forgot to add the changes to the file before amending the commit, kids, don't git while tired.